send_sms.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import json
+import general
+
+URL = 'http://www.way2sms.com/api/v1/sendCampaign'
+
+
+# get request
+def sendPostRequest(reqUrl, apiKey, secretKey, useType, phoneNo, senderId, textMessage):
```

Comment:
Most of the params for this method should be pulled in from the config file. 
Can we do something similar to https://github.com/jarifibrahim/resultBot/blob/master/send_email.py#L7 ?